jolly-jellyfish/src/static/themes/win95.css

```diff
@@ -3,6 +3,7 @@
  * A static file "/static/fonts/hellovetica.woff2" should exist,
  * but it will fall back on "News Cycle" from Google Fonts if not.
  * Static files "/static/images/win95-toolbar.png" and "/static/images/win95-close.png" should also exist.
+ * Requires that images be wrapped in divs.
```

Comment:
Mind writing what class is required?

Reply:
This theme isn't picky about the class, but I do need to put that in the Console and Galaxy themes.

This theme does still require the divs though, to make 100% relative to the size of the image.